source/help/settings/team-settings.md

```diff
@@ -17,6 +17,12 @@ Your **Team Name** is displayed on the sign-in page, and in the top of the left-
 
 You can enter a name up to 15 characters in length. The length of team names is limited to ensure readability.
 
+#### Team Description
+
+Your **Team Description** is displayed on the Team Selection page and in the tooltip when hovering over the team name in the team sidebar.
```

Comment:
Should we switch this to "join teams" page or something, now that we don't show that page on log in? 

Maybe "is displayed when viewing the list of teams available to join" or something like that